src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigureTwo implements AreaOfFigure, InfoFigure {
+    //прямоугольный треугольник
+    private double height; // высота
+
+    public RightTriangle(String color, double firsSide, double height) {
+        super(color, firsSide, height);
+    }
+
+    @Override
+    public double detAreaOfFigure() {
+        return (getFirsSide() * height) / 2;
+    }
+
+    @Override
+    public String infoFigure() {
+        return ("Высота = " + height + "см ,"
+                + "Ширина = " + getSecondSide() + "см ,"
+                + "Площадь треугольника = " + detAreaOfFigure()) + "см";
```

Comment:
Avoid using non-English text in your code. The checklist advises using only English in messages and code to maintain universality and consistency.